bedrock/firefox/views.py

```diff
@@ -473,6 +472,16 @@ def get_template_names(self):
                 template = "firefox/developer/whatsnew.html"
             else:
                 template = "firefox/whatsnew/index.html"
+        elif version.startswith("125."):
+            if locale in ["de", "en-GB", "es-ES", "fr", "it", "pl"]:
+                template = "firefox/whatsnew/whatsnew-fx125-eu.html"
+            elif locale in ["en-US", "en-CA"]:
```

Comment:
Shouldn't this be:
```suggestion
            elif locale in ["en-"]:
```
?

Reply:
We want to exclude en-GB since that's part of the rule above (and there's a separate rule for anyone using en-US or en-CA but geographically in Great Britain).